airbyte-integrations/connectors/source-google-sheets/metadata.yaml

```diff
@@ -13,7 +13,6 @@ data:
   name: Google Sheets
   registries:
     cloud:
-      dockerImageTag: 0.2.21
```

Comment:
Unpinning because this no longer needs to be pinned:
> [This PR added a field to the spec](https://github.com/airbytehq/airbyte/pull/23057/files#diff-7713475557366d1d512b0822754fe63fc44ea9fc3fd85d274f64965a741a1487), but it defaults to [false](https://github.com/airbytehq/airbyte/pull/23057/files#diff-7713475557366d1d512b0822754fe63fc44ea9fc3fd85d274f64965a741a1487R27), so the [default behavior will stay the same](https://github.com/airbytehq/airbyte/pull/23057/files#diff-c397937888441dc0f05d5a3a8590cabbaba287b0f305001c89ab72df9fc4100fR147-R148) :+1:
[The major bump is just a change in license, no change in behavior](https://github.com/airbytehq/airbyte/pull/27738/files) :+1:
I don't see [outstanding issues for source-google-sheets](https://github.com/airbytehq/airbyte/issues?q=is%3Aopen+is%3Aissue+label%3Aconnectors%2Fsource%2Fgoogle-sheets), so I expect the latest version of the connector to work as expected